pandas/tests/interchange/test_impl.py

```diff
@@ -287,9 +287,8 @@ def test_multi_chunk_pyarrow() -> None:
     names = ["n_legs"]
     table = pa.table([n_legs], names=names)
     with pytest.raises(
-        RuntimeError,
-        match="To join chunks a copy is required which is "
-        "forbidden by allow_copy=False",
+        pa.ArrowInvalid,
```

Comment:
I think we have generally been trying to catch the pyarrow error internally and raise something more generic from pandas but @mroeschke would know best

Reply:
Yes ideally this would be the case, but we're definitely not consistent about it

Reply:
sure, i'm catching to re-raise RunTimeError (which is what currently gets raised)